types/stripe-js/payment-intents.d.ts

```diff
@@ -90,6 +91,15 @@ declare module '@stripe/stripe-js' {
     };
   }
 
+  interface CreatePaymentMethodGrabPayData extends PaymentMethodCreateParams {
+    type: 'grabpay';
+
+    /**
+     * Can be omitted as there are no GrabPay-specific fields.
+     */
+    grabpay: {};
```

Comment:
Is this field optional? Or is it required but can be blank?
```suggestion
    grabpay?: {};
```

Reply:
Fixed!